superset-frontend/plugins/legacy-preset-chart-deckgl/src/components/Legend.tsx

```diff
@@ -106,6 +106,7 @@ const Legend = ({
       <li key={k}>
         <a
```

Comment:
Let's create a base component for links. When onClick is present we might assign the role button. Let's also account for the possibility to override the prop

Reply:
Per the note below, I believe we settled on leaving as is for now because this is a plugin 
https://github.com/apache/superset/pull/26602/files#r1504672422

Reply:
I agree otherwise we might have to face issues with circular dependencies here